dist/backdraft.js

```diff
@@ -845,11 +845,19 @@ _.extend(Plugin.factory, {
     ',
     sFilterListTemplate : '\
       <li><label><input class="list" id="value" type="checkbox" name="{{= attr }}" \
-         value="{{= options }}" /> <%= options %></label></li> \
+         value="{{= options }}" /> {{= options }}</label></li> \
+    ',
+    sActionButtonTemplate : '\
+      <button class="btn btn-primary btn-sm {{= klass}}" name="button" type="submit" title="">{{= caption}}</button>\
```

Comment:
I didn't catch this before but we don't do hungarian notation for variable names.. (dataTables does/did but I think their 1.10 got rid of it and for our interfaces we don't expose that).

Can you remove the leading `s` on these? 

What about a nested structure here? I.e.:

```
filterTemplates : {
  string : '...',
  numeric : '...',
  list : '...'
},
```

If the action button template for filters only? I think so far it is?

So could include in that nesting and clarify the other items as the input portion of the templates?

```
filterTemplates : {
  stringInput : '...',
  numericInput : '...',
  listInput : '...',
  actionButton : '...'
},
```